src/main/java/com/redhat/red/build/koji/model/xmlrpc/KojiWinBuildInfo.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc. (jcasey@redhat.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.red.build.koji.model.xmlrpc;
+
+import java.util.List;
+
+import org.commonjava.rwx.binding.anno.Converter;
+import org.commonjava.rwx.binding.anno.DataKey;
+import org.commonjava.rwx.binding.anno.StructPart;
+
+@StructPart
+public class KojiWinBuildInfo
```

Comment:
Again, it is no more optional than maven build support, which is enabled via an option:

`def mavenEnabled(self):
        return bool(context.opts.get('EnableMaven'))`

and

` def winEnabled(self):
        return bool(context.opts.get('EnableWin'))`

all of which fall under the header of "non-rpm support". But, the code is all mixed together.